<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>199</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>General settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="METEOVAR_LBL">
     <property name="text">
      <string>Select the variable to be deleted</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="METEOVAR_CB"/>
   </item>
   <item>
    <widget class="QLabel" name="WS_LBL">
     <property name="text">
      <string>Select the weather station name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="WS_CB"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="FROMLBL">
        <property name="text">
         <string>from:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="FROMDATE"/>
      </item>
      <item>
       <widget class="QLabel" name="TOLBL">
        <property name="text">
         <string>to:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="TODATE"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
